vllm/_custom_ops.py

```diff
@@ -232,8 +232,11 @@ def swap_blocks(src: torch.Tensor, dst: torch.Tensor,
     vllm_cache_ops.swap_blocks(src, dst, block_mapping)
 
 
-def convert_fp8(output: torch.Tensor, input: torch.Tensor) -> None:
-    vllm_cache_ops.convert_fp8(output, input)
+def convert_fp8(input: torch.Tensor,
```

Comment:
Nit: For these kinds of functions that have out parameters (like output), we have the convention to put the out parameter first (see the other functions in this file and also the CUDA functions)

Reply:
Got it. Will change.

Reply:
(this file is called a bunch from python code, so reversing the parameters would be a breaking change for these as well)

Reply:
@comaniac Actually I see now that this was just a bug and the underlying function takes input first -- in that case I think what you did is fine (we probably shouldn't change it as part of this PR), sorry for the noise :)

Reply:
No worries. I changed this to align its CUDA function signature. I can do it in a reverse way if there's anything else I need to change.